### ISSUE TITLE
[format-js] Define reprint code style

### ISSUE THREAD

**Issue by DaleJefferson** [facebook/nuclide#328]
I'm looking for a code formatter that supports Flow and nuclide-format-js "Reprint all code" seems to be the closest thing I can find. The initial problem with this approach is that I can't see anyway to set the code style.

Tab width, Branch style etc...

**Comment by johnislarry** [comment]
cc @kyldvs who wrote format-js.

**Comment by kyldvs** [comment]
Yeah, it is not configurable at all yet. The pretty printer is a very experimental feature.

First I'm focused on having it break things according to a single style guide before allowing custom styles to be substituted in for different AST nodes.
